great_expectations/expectations/expectation.py

```diff
@@ -984,7 +992,7 @@ def get_result_format(
         configuration: ExpectationConfiguration,
         runtime_configuration: dict = None,
     ) -> Union[Dict[str, Union[str, int, bool]], str]:
-        default_result_format: Optional[str] = self.default_kwarg_values.get(
+        default_result_format: Optional[Any] = self.default_kwarg_values.get(
```

Comment:
Constraining `default_kwarg_values` from `Any` to `object` meant that this change was required. After reading `mypy` documentation, I'm not sure I understand why though.